spirv_cross_c.h

```diff
@@ -336,7 +336,7 @@ typedef struct spvc_msl_vertex_attribute
  */
 SPVC_PUBLIC_API void spvc_msl_vertex_attribute_init(spvc_msl_vertex_attribute *attr);
 
-/* Maps to C++ API. */
+/* Maps to C++ API. Deprecated; use spvc_msl_shader_interface_var_2. */
```

Comment:
When updating the C API headers, we also need to update

`SPVC_C_API_VERSION_MINOR` and spirv-cross-abi-minor in CMakeLists.txt.

Reply:
Oops. Done.